### PR TITLE
CN0338 CLI fix

### PR DESCRIPTION
**Pull request by adrimbarean**
Fix the CLI functionality by:
 - correcting bug in the ring buffer management;
 - increasing the rx buffer size;
 - removing error throw in valid application branch.

### CODE REVIEW

**Comment by adrimbarean** [comment]
V0 -> V1:
 - fix travis-ci errors (astyle).

File: projects/ADuCM360_demo_cn0338/src/Cmd.cpp [hunk 1]
```diff
@@ -222,6 +222,10 @@ void Cmd_ReadData(void)
       char c = uart_rx_queue[uart_rx_head];
       if (c >= ' ' && c <= '~') {
          handle_ascii_data(c);
+         if(uart_rx_head == 63)
```
Comment (mirceacaprioru) [inline projects/ADuCM360_demo_cn0338/src/Cmd.cpp]: Do you have a define for RING_BUFFER_LEN ? I would use that. If you encrease the size of the buffer you will have to update here manually also.

Comment (mirceacaprioru) [reply projects/ADuCM360_demo_cn0338/src/Cmd.cpp]: I think this is the define to use UART_RX_QUEUE_SIZE

**Comment by mirceacaprioru** [comment]
Just a small comment. Rest looks good.

**Comment by adrimbarean** [comment]
V1 -> V2;
 - use UART_RX_QUEUE_SIZE instead of hard-coded number

**Pull request merged**